### PR TITLE
Update haproxy_apache_lxd.md (punctuation, grammar, ...)

### PR DESCRIPTION
**Pull request by SergeCroise**
- fully qualified ... (FQDN)
- exit the container
- This can be done ...

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

**Comment by SergeCroise** [comment]
Hello @sspencerwire , @ambaradan ,
please review!
(feel free to overwrite my changes)

File: docs/guides/proxies/haproxy_apache_lxd.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tested_with: 8.5, 8.6, 9.0
 
 ## Introduction
 
-HAProxy stands for "High Availability Proxy." This proxy can sit in front of any TCP application (such as web servers), but it is often used to act as a load-balancer between multiple instances of a website.
+HAProxy stands for "High Availability Proxy". This proxy can sit in front of any TCP application (such as web servers), but it is often used to act as a load-balancer between multiple instances of a website.
```
Comment (sspencerwire) [inline docs/guides/proxies/haproxy_apache_lxd.md:11]: I actually think that the period should be inside the quotes here. It's not a code block, which would be the only exception I can think of. See this: https://docs.rockylinux.org/guides/contribute/style_guide/#grammar-and-punctuation and the bullet on periods and commas.

**Comment by sspencerwire** [comment]
@SergeCroise This looks good except for that one item. Please review and update that and I'll get this merged. Thank you!!

**Review by wsoyinka — APPROVED** [review]
Thank you @SergeCroise :-)

**Pull request merged**